<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PBIDetailsWidget</class>
 <widget class="QWidget" name="PBIDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="IndicatorWidget" name="pbiDetailsIndicator" native="true"/>
   </item>
   <item>
    <widget class="QStackedWidget" name="pbiUpdateStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_7">
      <layout class="QVBoxLayout" name="verticalLayout_17">
       <item>
        <widget class="QTreeWidget" name="pbiUpdateList">
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Installed version</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New version</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="updateSelectedPBIBtn">
           <property name="text">
            <string>Update selected only</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/install.png</normaloff>:/images/install.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_8">
      <layout class="QVBoxLayout" name="verticalLayout_19">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Update log:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="pbiUpdateLog">
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IndicatorWidget</class>
   <extends>QWidget</extends>
   <header location="global">indicatorwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
